mcos/error_estimator.py

```diff
@@ -15,3 +15,17 @@ def estimate(self, mu: np.array, cov: np.array, allocation: np.array, optimal_al
         @return
         """
         pass
+
+
+class ExpectedOutcomeErrorEstimator(AbstractErrorEstimator):
+    """Error Estimator that calculates the mean difference in expected outcomes"""
+
+    def estimate(self, mu: np.array, cov: np.array, allocation: np.array, optimal_allocation: np.array) -> float:
+        return np.dot(optimal_allocation - allocation, mu)
+
+
+class VarianceErrorEstimator(AbstractErrorEstimator):
+    """Error Estimator that calculates the mean difference in variance"""
+
+    def estimate(self, mu: np.array, cov: np.array, allocation: np.array, optimal_allocation: np.array) -> float:
+        return np.dot(optimal_allocation - allocation, np.dot(optimal_allocation - allocation, cov))
```

Comment:
This calculates the
<img width="486" alt="Screen Shot 2020-01-29 at 2 11 28 PM" src="https://user-images.githubusercontent.com/24689747/73388990-3f74ab80-42a1-11ea-8aa9-623199277444.png">
where the first `w` is the optimal allocation, the second `w` is the allocation and the `V` is the covariance matrix.

Reply:
Could you change the latter part to `np.dot(cov, optimal_allocation - allocation)`? I think the order matters.

Reply:
Updated